### PR TITLE
Working Tests and a few pre-work Methods for LeagueStat

### PR DESCRIPTION
**Pull request by LHJE**

### CODE REVIEW

File: lib/league_stats.rb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+  def load_game_teams(filepath)
+    CSV.foreach(filepath, headers: true, header_converters: :symbol) do |data|
+      @game_teams_stuff << GameTeams.new(data)
+    end
+  end
+
+  def find_by_game_id(game_id)
+    @game_teams_stuff.find do |season_stat|
+      # require 'pry'; binding.pry
```
Comment (BJSherman80) [inline lib/league_stats.rb:28]: Remove Pry

File: lib/league_stats.rb [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  #  # Name of the team with the lowest average score per game across all seasons when they are a visitor.  STRING
+  #   def lowest_scoring_visitor
+  #
+  #   end
+  #
+  #  # 	Name of the team with the lowest average score per game across all seasons when they are at home.  STRING
+  #   def lowest_scoring_home_team
+  #
+  #   end
+
```
Comment (BJSherman80) [inline lib/league_stats.rb:68]: Great job at pseudo coding !!

**Review by BJSherman80 — APPROVED** [review]

File: lib/league_stats.rb (hunk 1 above)
Comment (LHJE) [reply lib/league_stats.rb:28]: Awesome, thank you!

File: lib/league_stats.rb (hunk 2 above)
Comment (LHJE) [reply lib/league_stats.rb:68]: Thank you!

**Pull request merged**